kitchen.yml

```diff
@@ -1,32 +1,107 @@
 ---
-driver:
-  name: vagrant
-
-## The forwarded_port port feature lets you connect to ports on the VM guest via
-## localhost on the host.
-## see also: https://www.vagrantup.com/docs/networking/forwarded_ports
-
-#  network:
-#    - ["forwarded_port", {guest: 80, host: 8080}]
-
 provisioner:
   name: chef_zero
-
-  ## product_name and product_version specifies a specific Chef product and version to install.
-  ## see the Chef documentation for more details: https://docs.chef.io/workstation/config_yml_kitchen/
-  #  product_name: chef
-  #  product_version: 17
+  enforce_idempotency: true
+  multiple_converge: 2
+  deprecations_as_errors: true
 
 verifier:
   name: inspec
 
-platforms:
-  - name: ubuntu-20.04
-  - name: centos-8
-
 suites:
-  - name: default
+  - name: osl_authorized_keys
+    run_list:
+      - recipe[osl-resources-test::osl_authorized_keys]
     verifier:
-      inspec_tests:
-        - test/integration/default
-    attributes:
+      controls:
+        - osl_authorized_keys
```

Comment:
Is this needed?

Reply:
nope i was experimenting with controls file structure and this was legacy, will remove